### PR TITLE
feat: warn when the babel preset is not configured

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: src/sheet.js [hunk 1]
```diff
@@ -49,6 +49,10 @@ function sheet() {
     throw new Error('Unable to insert stylesheet');
   }
 
+  if (process.env.NODE_ENV !== 'test') {
```
Comment (thymikee) [inline src/sheet.js]: you need to test for `process`, afaik browser will throw evaluating this code.

Comment (satya164) [reply src/sheet.js]: it doesn't

Comment (zamotany) [reply src/sheet.js]: When prevaling the code NODE_ENV won't be `test`

Comment (zamotany) [reply src/sheet.js]: you should check for typeof window i guess

Comment (satya164) [reply src/sheet.js]: @zamotany this code will only execute in browser or jsdom since there is a check for `document` above

**Comment by thymikee** [comment]
Looks like you need to add a test :D

**Comment by satya164** [comment]
Sorry, I'm not able to add a test because the environment is always `test` on travis even if I assign `process.env.NODE_ENV` to a different value. I'll merge this for now so I can do a release and add a test tomorrow.

**Comment by thymikee** [comment]
Just remove this console.log

File: src/__tests__/__snapshots__/sheet.browser.spec.js.snap [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`sheet module (browser) should warn for configuring babel preset 1`] = `"Babel preset for Linaria is not configured. See https://github.com/callstack-io/linaria/blob/master/docs/BABEL_PRESET.md for instructions."`;
```
Comment (thymikee) [inline src/__tests__/__snapshots__/sheet.browser.spec.js.snap]: There's a snapshot but no test

**Pull request merged**